Lib/tkinter/test/test_tkinter/test_widgets.py

```diff
@@ -307,6 +307,10 @@ class OptionMenuTest(MenubuttonTest, unittest.TestCase):
     def create(self, default='b', values=('a', 'b', 'c'), **kwargs):
         return tkinter.OptionMenu(self.root, None, default, *values, **kwargs)
 
+    def test_bad_kwarg(self):
+        with self.assertRaisesRegex(TclError, r"^unknown option -image$"):
+            tkinter.OptionMenu(self.root, None, 'b', 'a', 'b', 'c', image='::tk::icons::question')
```

Comment:
Remove irrelevant and distracting detail from call.
```suggestion
            tkinter.OptionMenu(self.root, None, 'b', image='')
```